package.json

```diff
@@ -46,77 +46,77 @@
   },
   "dependencies": {
     "csstype": "^2.6.0",
-    "deep-object-diff": "^1.0.4",
+    "deep-object-diff": "^1.1.0",
     "emotion": "^8.0.8",
-    "history": "^4.6.1",
-    "lodash": "^4.17.4",
-    "marksy": "^6.0.2",
-    "normalize.css": "^7.0.0",
-    "prismjs": "1.6.0",
+    "history": "^4.7.2",
+    "lodash": "^4.17.11",
+    "marksy": "^6.1.0",
+    "normalize.css": "^8.0.1",
+    "prismjs": "1.15.0",
     "react-emotion": "^8.0.8",
     "react-live": "^1.12.0",
-    "react-redux": "^5.0.5",
+    "react-redux": "^5.1.1",
     "react-transition-group": "1.2.1",
-    "react-typography": "^0.16.5",
-    "redux": "^3.0.4",
-    "redux-actions": "^2.0.3",
+    "react-typography": "^0.16.18",
+    "redux": "^4.0.1",
+    "redux-actions": "^2.6.4",
     "to-style": "^1.3.3",
-    "victory-core": "^15.2.0"
+    "victory-core": "^31.1.0"
   },
   "peerDependencies": {
-    "prop-types": "^15.6.0",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0"
+    "prop-types": "^15.6.2",
+    "react": "^16.7.0",
```

Comment:
Do we want to require 16.7 above? Or can we support anything below that? Ideally all `peerDependencies` are as permissive as possible to allow folks to use this package that aren't at the latest version if everything is still correct.

Same for all other deps here.

Reply:
okay - will make the latest major version the base-level requirement for peer deps, which I think is reasonable!